s3tests_boto3/functional/__init__.py

```diff
@@ -205,6 +207,12 @@ def setup():
     config.tenant_user_id = cfg.get('s3 tenant',"user_id")
     config.tenant_email = cfg.get('s3 tenant',"email")
 
+    config.iam_access_key = cfg.get('iam',"access_key")
+    config.iam_secret_key = cfg.get('iam',"secret_key")
+    config.iam_display_name = cfg.get('iam',"display_name")
+    config.iam_user_id = cfg.get('iam',"user_id")
+    #config.iam_email = cfg.get('iam',"email")
```

Comment:
if this line is commented out, can you remove it?

Reply:
I'm not using this but if someone wants then they can use it that's the reason why I thought keeping it.

Reply:
If there's no use for it right now in any of the tests could you remove it?